src/VirtoCommerce.Platform.Core/PlatformConstants.cs

```diff
@@ -67,7 +67,7 @@ public static class Settings
         {
             public static class Security
             {
-                public static SettingDescriptor SecurityAccountTypes = new SettingDescriptor
+                public static SettingDescriptor SecurityAccountTypes { get; set; } = new SettingDescriptor
```

Comment:
left only `get;`  accessors for constants.

Reply:
OK